src/MIDebugEngine/Natvis.Impl/Natvis.cs

```diff
@@ -191,16 +191,44 @@ public void Initialize(string fileName)
             {
                 // failed to find the VS Service
             }
-            string natvisPath = null;
-            string directory = _process.Engine.GetMetric("GlobalVisualizersDirectory") as string;
-            if (!string.IsNullOrEmpty(directory) && !string.IsNullOrEmpty(fileName))
+
+            if (string.IsNullOrEmpty(Path.GetDirectoryName(fileName)))
```

Comment:
This will return true if `fileName` is something like `folder\name.ext`. Seems like this should be Path.IsRooted instead.